install/kubernetes/helm/istio/values-istio-sds-fallback-file-mount-test.yaml

```diff
@@ -0,0 +1,22 @@
+global:
```

Comment:
Where is this file `values-istio-sds-fallback-file-mount-test.yaml` used? is it only for test purpose?